src/main/java/org/candlepin/async/tasks/CertificateCleanupJob.java

```diff
@@ -44,6 +44,7 @@ public class CertificateCleanupJob implements AsyncJob {
 
     public static final String JOB_KEY = "CertificateCleanupJob";
     public static final String JOB_NAME = "Certificate Cleanup";
+    // Every noon
```

Comment:
Quartz uses a bit different format for schedule. It starts with seconds instead of minutes, so the third value specifies hours and fourth would specify days.

Reply:
Oh interesting!! Thanks for clarifying that